services/src/test-library/src/main/java/org/openkilda/testing/service/northbound/NorthboundServiceImpl.java

```diff
@@ -93,6 +94,27 @@ public FlowPayload getFlow(String flowId) {
                 new HttpEntity(buildHeadersWithCorrelationId()), FlowPayload.class, flowId).getBody();
     }
 
+    @Override
+    public List<FlowEventPayload> getFlowHistory(String flowId) {
+        return Arrays.asList(restTemplate.exchange("/api/v1/flows/{flow_id}/history", HttpMethod.GET,
```

Comment:
Looks like you can write `return getFlowHistory(flowId, null, null);` here.

Reply:
Done.